ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Ledger/Tables.hs

```diff
@@ -8,12 +8,26 @@
 {-# LANGUAGE TypeOperators            #-}
 {-# LANGUAGE UndecidableInstances     #-}
 
--- | See @'LedgerTables'@
+-- | This module defines the 'LedgerTables', a portion of the Ledger notion of a
+-- /ledger state/ (not to confuse with our
+-- 'Ouroboros.Consensus.Ledger.Basics.LedgerState') that together with it,
+-- conforms a complete Ledger /ledger state/.
+--
```

Comment:
Do you know if we have a good explanation on why we call this part of the ledger state "tables"? Are we making use of the database jargoon?

Reply:
I am pretty sure we use the database jargoon. however this comes from the db-api pdf report